### PR TITLE
Proposal for improvements for ui5-proxy

### PR DESCRIPTION
**Pull request by tobiasqueck**
## Description
@zdravko-georgiev @cfg74 , as discussed, here are the improvements for the `ui5-proxy-middleware`.

* use `debug` logging for the logging of forwarded files because it is otherwise so chatty that you don't see any other logs from other middlewares
* use a default setting for ui5 so that for the default case nothing needs to be provided as config in the yaml
```js
{
    url: 'https://ui5.sap.com',
    path: ['/resources', '/test-resources']
}
```

## Changes
* fix for https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&types=BUG&id=SAP_open-ux-tools&open=AYfrPkWISN0lrgwATVQm
* changed log level for each loaded file from `info` to `debug`
* set default config if none is provided
* updated the readme

### CODE REVIEW

File: packages/ui5-proxy-middleware/README.md [hunk 1]
```diff
@@ -5,15 +5,27 @@ The `@sap-ux/ui5-proxy-middleware` is a [Custom UI5 Server Middleware](https://s
 ## Configuration Options
 | Option       | Default Value | Description |
 | ------------ | ------------- | ----------- |
-| `ui5`        | mandatory     | List of mount paths and target urls that should be handled by the proxy |
+| `ui5`        | `object`      | List of mount paths and target urls that should be handled by the proxy. If not provided then `/resources` and `/test-resources` are proxied to `https://ui5.sap.com` |
```
Comment (devinea) [inline packages/ui5-proxy-middleware/README.md:8]: for `ui5-local.yaml` the proxy is configured without the ui5 option. Won't this default now be applied which wouldn't really be expected in offline mode.

File: packages/ui5-proxy-middleware/src/ui5/middleware.ts [hunk 2]
```diff
@@ -115,14 +116,14 @@ module.exports = async ({ resources, options }: MiddlewareParameters<UI5ProxyCon
...
-        const directLoadProxy = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+        const directLoadProxy = (async (req: Request, res: Response, next: NextFunction): Promise<void> => {
             try {
                 await injectScripts(req, res, next, ui5Configs);
             } catch (error) {
                 logger.error(error);
                 next(error);
             }
-        };
+        }) as RequestHandler;
```
Comment (devinea) [inline packages/ui5-proxy-middleware/src/ui5/middleware.ts:126]: Thanks for the sonar fix.

File: packages/ui5-proxy-middleware/README.md (hunk 1 above)
Comment (tobiasqueck) [reply packages/ui5-proxy-middleware/README.md:8]: @devinea this default is only used when using the open source `ui5-proxy-middleware` directly in the yaml. The integrated version in `@sap/ux-ui5-tooling` is still in control of the used settings. And, yes, you are completely correct, for backward compatibility the `fiori-tools-proxy` can sadly not introduce this as default

Comment (devinea) [reply packages/ui5-proxy-middleware/README.md:8]: OK. Thanks for clarification.

**Review by devinea — APPROVED** [review]
Changset ✅ 
New defaults seem like sensible approach.
Code changes and tests look good. Coverage is excellent.
Fix for code smell in main ✅

**Review by zdravko-georgiev — APPROVED** [review]
Code is well structured and easy to understand
Excellent test coverage
Didn't tested manually

**Pull request merged**